### PR TITLE
Bugfix script option {sRemotePathFromUrl}

### LINKED ISSUE #6567: incorrect {sRemotePathFromUrl} parameter evaluation
## Current behaviour
the script `{sRemotePathFromUrl}` variable evaluation is incorrect

```
sRemote=origin
sRemoteUrl=git@gitlab-be.mtrs.intl:ASW/Focus_gitlabCI_test.git
sRemotePathFromUrl=git@gitlab-be.mtrs.intl:ASW/Focus_gitlabCI_test.git
```

## Expected behaviour
`{sRemotePathFromUrl}` should evaluate to `/ASW/Focus_gitlabCI_test` similar to  `{cDefaultRemotePathFromUrl}`
```
cDefaultRemote=origin
cDefaultRemoteUrl=git@gitlab-be.mtrs.intl:ASW/Focus_gitlabCI_test.git
cDefaultRemotePathFromUrl=/ASW/Focus_gitlabCI_test
```

## Steps to reproduce
create a script
```
command = echo
arguments = cDefaultRemotePathFromUrl = {cDefaultRemotePathFromUrl}
sRemotePathFromUrl = {sRemotePathFromUrl}
```

create a branch,check it out and push it to your upstream repo `origin`

select your branch commit and select the script from the context menu

## Did this work in previous version of GitExtensions

this worked as expected in V3.0.2

## Environment
- Git Extensions 3.1.0.5877
- Build bb1687f6bd0450d0c1a2bae10a37c65493fc1294
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3362.0
- DPI 96dpi (no scaling)

### PR DESCRIPTION
**Pull request by mstv**
Fixes #6567

## Proposed changes

- Complete the refactoring done in #6004 for sRemotePathFromUrl

## Test methodology 

- added NUnit tests

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by mstv** [comment]
@jeroendv, could you test the [canary build 5886](https://ci.appveyor.com/api/buildjobs/3rjvvd3katvhm92u/artifacts/Setup%2FGitExtensions-Portable-3.2.0.5886.zip), please?

File: GitUI/Script/ScriptOptionsParser.cs [hunk 1]
```diff
@@ -349,7 +349,8 @@ private static string ParseScriptArguments(string argument, string option, IWin3
                         remote = selectedRemotes.Count == 1
                             ? selectedRemotes[0]
                             : AskToSpecify(selectedRemotes, revisionGrid);
-                        newString = module.GetSetting(string.Format(SettingKeyString.RemoteUrl, remote));
+                        url = module.GetSetting(string.Format(SettingKeyString.RemoteUrl, remote));
+                        newString = GetRemotePath(url);
```
Comment (RussKie) [inline GitUI/Script/ScriptOptionsParser.cs]: Can we add a test for this?

**Comment by jeroendv** [comment]
> @jeroendv, could you test the [canary build 5886](https://ci.appveyor.com/api/buildjobs/3rjvvd3katvhm92u/artifacts/Setup%2FGitExtensions-Portable-3.2.0.5886.zip), please?

I can confirm that the bug is fixed.

**Comment by mstv** [comment]
> I can confirm that the bug is fixed.

Thank you.

File: GitUI/Script/ScriptOptionsParser.cs (hunk 1 above)
Comment (mstv) [reply GitUI/Script/ScriptOptionsParser.cs]: I'll add some tests but not for every script option. (We should have asked for tests before the `ScriptOptionParser` was refactored. This is at least the second bug caused by this refactoring.)

**Comment by RussKie** [comment]
I don't expect test for the whole parser (unless you want to). Just for the
immediate change.
Thank you

**Pull request merged**